### ISSUE TITLE
Stuck with Central install

### ISSUE THREAD

**Issue by JRowswell** [talosh/flameTimewarpML#93]
Hey,

Probably just my lack of knowledge/misreading but I can't seem to run this command in the central install instructions: 
/mnt/software/flameTimewarpML/init_env  /mnt/software/miniconda3/           # Linux

I get this error:
-bash: /mnt/software/flameTimewarpML/init_env: No such file or directory

I've run through all the instructions successfully so far (no errors) so not sure what I am doing wrong.
Any help would be greatly appreciated!

Cheers

**Comment by JRowswell** [comment]
Still unable to test

Is it possible the init_env hasn't been shipped with the bundled file?

Help would be appreciated. 

Cheers

**Comment by talosh** [comment]
Would you like to give v0.4.5 a try? It is in active development and should
not need anything special for centralized installation, just to place a
folder into a python hooks path should be enough.

**Comment by JRowswell** [comment]
Sounds good! 

Cheers

**Comment by JRowswell** [comment]
Does this mean when I install it on more than one machine it won't ask to unpack each time? 

Cheers

**Comment by talosh** [comment]
Yes there’s no packed python environment, it just lives in packages folder and avalable straight away.Sent from my iPhoneOn 30 May 2024, at 08:20, JRowswell ***@***.***> wrote:﻿
Does this mean when I install it on more than one machine it won't ask to unpack each time?
Cheers

**Comment by talosh** [comment]
There is v0.4.5 dev 003 in releases to test, would you mind to provide any feedback on if it is works for you as a centralized install for many machines?
